docs/json/radarr/cf/anime-dual-audio.json

```
@@ -9,7 +9,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "dual[ ._-]?audio|[\\[(]dual[\\])]|(JA|ZH)\\+EN|EN\\+(JA|ZH)"
+        "value": "dual[ ._-]?audio|[\\[(]dual[\\])]|(?-i)DUAL-VARYG(?i)|dual[ ._-]?(\\d{3,4}p|ultrahd|4k)|(\\d{3,4}p|ultrahd|4k)[ ._-]?dual|(JA|ZH)\\+EN|EN\\+(JA|ZH)"
```

I'm not sure these test cases are thorough enough to really tell, but given that regex is case insensitive, the one match that is made for lowercase dual-varyg would happen anyway I from what nux said.

https://regex101.com/r/wojuc0/1

```suggestion
        "value": "[\\[(]?(dual)([\\s._\\])-]?(audio|varyg|\\d{3,4}p|ultrahd|4k))|(\\d{3,4}p|ultrahd|4k)[\\s._-]?dual|(JA|ZH)\\+EN|EN\\+(JA|ZH)
```